deploy/helm/sumologic/values.yaml

```
@@ -970,6 +970,21 @@ prometheus-operator:
         ## kube_node_status_allocatable
         ## kube_node_status_capacity
         ## kube_node_status_condition
+        ## kube_statefulset_metadata_generation
+        ## kube_statefulset_replicas
+        ## kube_statefulset_status_observed_generation
+        ## kube_statefulset_status_replicas
+        - url: http://$(CHART).$(NAMESPACE).svc.cluster.local:9888/prometheus.metrics.state
+          writeRelabelConfigs:
+            - action: keep
+              regex: kube-state-metrics;(?:kube_statefulset_status_observed_generation|kube_statefulset_status_replicas|kube_statefulset_replicas|kube_statefulset_metadata_generation|kube_daemonset_status_current_number_scheduled|kube_daemonset_status_desired_number_scheduled|kube_daemonset_status_number_misscheduled|kube_daemonset_status_number_unavailable|kube_deployment_spec_replicas|kube_deployment_status_replicas_available|kube_deployment_status_replicas_unavailable|kube_node_info|kube_node_status_allocatable|kube_node_status_capacity|kube_node_status_condition)
+              sourceLabels: [job, __name__]
+            - action: labelmap
+              regex: (pod|service)
+              replacement: sd_${1}
```

I think we should go further and use `service_discovery_` - no need to use the non-obvious `sd_` abbreviation.
```suggestion
              replacement: service_discovery_${1}
```